content/docs/2.13/authentication-providers/aws.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "Pod Identity Webhook for AWS"
++++
+
+[**AWS IRSA Pod Identity Webhook**](https://github.com/aws/amazon-eks-pod-identity-webhook), which is described more in depth [here](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/), allows you to provide the role name using an annotation on a service account associated with your pod.
+
+You can tell KEDA to use AWS Pod Identity Webhook via `podIdentity.provider`.
+
+```yaml
+podIdentity:
+  provider: aws # Optional. Default: none
+  roleArn: <role-arn|workload> # Optional. Default: RoleArn from annotation on service-account.
+```
+
+AWS IRSA will give access to pods with service accounts having appropriate annotations. Refer
```

Comment:
I think we should clarify (better) with the different roleArn options are

Reply:
There is only really one when using IRSA, the gist of it being that `eks.amazonaws.com/role-arn` should be set as an annotation on the service account which requires access to AWS services.

I added a HOWTO in argoproj/argo-cd#14187, perhaps a sequence of steps clarifying how to setup IRSA + KEDA would be useful here as well?

Reply:
+1000

Reply:
I'll check it tomorrow for adding it

Reply:
done